scripts/update/Updater.php

```diff
@@ -283,6 +285,22 @@ public function update($initialVersion) {
 
             $this->setVersion('1.9.1');
         }
+
+        if ($this->isVersion('1.9.1')) {
+            $assignmentService = new ProctoringAssignmentService();
+            $assignmentService->setServiceManager($this->getServiceManager());
+            $this->getServiceManager()->register(ProctoringAssignmentService::CONFIG_ID, $assignmentService);
+
+            $deliveryExt = \common_ext_ExtensionsManager::singleton()->getExtensionById('taoDelivery');
+            $deliveryServerConfig = $deliveryExt->getConfig('deliveryServer');
+            $deliveryServerOptions = $deliveryServerConfig->getOptions();
+
+            $deliveryServerService = new DeliveryServerService($deliveryServerOptions);
+            $deliveryServerService->setServiceManager($this->getServiceManager());
+            $this->getServiceManager()->register(DeliveryServerService::CONFIG_ID, $deliveryServerService);
+
+            $this->setVersion('1.9.2');
+        }
     }
```

Comment:
@hutnikau @jsconan This is an update script too complex for a fix, this upgrade should have been a feature to 1.10